code/game/objects/items/storage/internal.dm

```diff
@@ -9,7 +9,6 @@
 		CRASH("Internal storage was created without a valid master object! ([loc], [usr])")
 	master_object = loc
 	name = master_object.name
-	verbs -= /obj/item/verb/verb_pickup	//make sure this is never picked up.
```

Comment:
any way to rectify this via a check or whatever?

Reply:
their locs will never be a turf, so yah it's no issue